id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/validator/IdRequestValidator.java

```diff
@@ -313,16 +305,37 @@ public void validateRequest(Object request, Errors errors, String method) {
 	 * @throws InvalidIdSchemaException
 	 */
 	@SuppressWarnings("unchecked")
-	private void validateVerifiedAttributes(Map<String, Object> requestMap)
+	private void validateVerifiedAttributes(Map<String, Object> requestMap,Errors errors)
 			throws IdObjectValidationFailedException, IdObjectIOException, InvalidIdSchemaException {
 		if (requestMap.containsKey(VERIFIED_ATTRIBUTES) && Objects.nonNull(requestMap.get(VERIFIED_ATTRIBUTES))
 				&& requestMap.get(VERIFIED_ATTRIBUTES) instanceof Map
 				&& !((Map<String, Object>) requestMap.get(VERIFIED_ATTRIBUTES)).isEmpty()
 				&& requestMap.containsKey(ROOT_PATH) && Objects.nonNull(requestMap.get(ROOT_PATH))
 				&& !((Map<String, Object>) requestMap.get(ROOT_PATH)).isEmpty()) {
-			String idSchema = restTemplate.getForObject(verifiedAttributesSchemaUrl, String.class);
+			String idSchema =getVerifiedAttributeIdSchema(verifiedAttributesSchemaUrl);
 			Map<String, Object> verifiedAttributesMap = (Map<String, Object>) requestMap.get(VERIFIED_ATTRIBUTES);
 			idObjectSchemaValidator.validateIdObject(idSchema, verifiedAttributesMap, verifiedAttributesFields);
+			Set<String> compositeKeySet = new HashSet<>();
+
+			for (Map.Entry<String, Object> entry : verifiedAttributesMap.entrySet()) {
+				String key = entry.getKey();
+				Object value = entry.getValue();
+				if (value instanceof List) {
+					List<Map<String, Object>> attributeList = (List<Map<String, Object>>) value;
+
+					for (Map<String, Object> attribute : attributeList) {
+						String trustFramework = (String) attribute.get("trustFramework");
+						String processName = (String) attribute.get("processName");
```

Comment:
We should not hardcode these fieldIds, instead it should be taken from properties.

Reply:
taken care